addons/test_mail/tests/test_performance.py

```diff
@@ -832,7 +832,7 @@ def _patched_unlink(records):
             unlinked_mails |= set(records.ids)
         unlinked_mails = set()
 
-        with self.assertQueryCount(admin=43, employee=43), \
+        with self.assertQueryCount(admin=55, employee=55), \
```

Comment:
Tha's... quite much, is there any issue with the cache like reading the server each time we send any email ? Or additional reads on attachments ?